### PR TITLE
Porting the existing visualization gallery 

### PR DESCRIPTION
**Pull request by nhintruong**
In this pull request, I port the 2nd and 4th visualization gallery from the NAU rcdata server

### CODE REVIEW

**Comment by Faye-yufan** [comment]
Hi @nhintruong , Could you please provide more specific details about what you're aiming to achieve in this PR?

In your proposal, you mentioned:
_Porting the gallery from the NAU rcdata server, to the new GitHub Pages gallery. In this tasks,
I will write an automatic script to download R codes linked to examples on NAU rcdata server,
re-run these codes with latest R, and adding new visualization to Github Pages. Finally, I will
update the repo.txt with the new added visualizations. In addition, I will revise the code of samples
in gallery to ensure it can work with latest R._

Is this what you plan to address in this PR?

**Comment by tdhock** [comment]
hi @nhintruong thanks for sharing
please click Files Changed tab, which shows what we see when we review your PR.
I see every line of meta.csv changed, but no significant changes, which is confusing, because you wrote "port the 2nd and 4th visualization" which made me think that only 2 lines would change.

rcdata gallery looks like below
image

Row 2 from rcdata "[Basis expansion for continuous neural network predictions](https://github.com/tdhock/2023-08-deep-learning/blob/main/slides/09-regression.R)" is already present in the github gallery as [tdhock/animint-figure-nnet-regression-degrees](https://github.com/tdhock/animint-figure-nnet-regression-degrees/tree/gh-pages) so it does not need to be ported again.

Row 4 from rcdata [Gradient descent for neural network and linear model](https://github.com/tdhock/animint-book/blob/7e7a224c004843d42a6f7e6caa30e319bcdce901/Ch18-neural-networks-sim.R) is not present in the github gallery and so could be ported.

Please make sure to double check which data viz have already been ported to the github gallery, so you do not duplicate any work that has already been done.

**Comment by tdhock** [comment]
also please submit future PRs from branches of animint/gallery instead of your fork (are you allowed?)

**Comment by nhintruong** [comment]
@tdhock It looks like I don't have write access to animint/gallery. I saw your invitation to animint organization and I accepted it but I still can't create a new branch. Could you please check my permission to this repository? 

image

**Comment by tdhock** [comment]
hi sorry about the permission issue, does it work now?
I added you to a new team with Maintain permissions.
